### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adarshM84**
New extention contribution

### CODE REVIEW

**Comment by jmorganca** [comment]
Thanks for the PR! Possible to limit it to one section?

**Comment by adarshM84** [comment]
> Thanks for the PR! Possible to limit it to one section?
Hii I have made changes.
You can add it in ### Extensions & Plugins section.

Thanks for the update.

File: README.md [hunk 1]
```diff
@@ -522,6 +522,7 @@ See the [API documentation](./docs/api.md) for all endpoints.
 - [AI Summmary Helper plugin](https://github.com/philffm/ai-summary-helper)
 - [TextCraft](https://github.com/suncloudsmoon/TextCraft) (Copilot in Word alternative using Ollama)
 - [Alfred Ollama](https://github.com/zeitlings/alfred-ollama) (Alfred Workflow)
+- [TextLLaMA](https://github.com/adarshM84/TextLLaMA) A Chrome Extension that Helps You Write Emails, Correct Grammar, and Translate into Any Language with a Few Fingertips! 🌟✉️
```
Comment (jmorganca) [inline README.md:525]:
```suggestion
- [TextLLaMA](https://github.com/adarshM84/TextLLaMA) A Chrome Extension that helps you write emails, correct grammar, and translate into any language
```

**Review by jmorganca — APPROVED** [review]
Very cool tool. Thanks!

**Pull request merged**